analysis/druidguardian/src/modules/features/GalacticGuardian.js

```diff
@@ -75,7 +76,7 @@ class GalacticGuardian extends Analyzer {
     const unusedGGProcs = 1 - (this.consumedGGProc / this.GGProcsTotal);
 
     return (
-      <StatisticBox
+      <Statistic
```

Comment:
I fixed the stats code for each of the modules I changed and got them in working order.  The logic in them needs to be updated as it is showing erroneous numbers (just from my in line math) so that will be another task after a transition to TS.